Runtime/Scripts/DataCollection/RGDataCollection.cs

```diff
@@ -0,0 +1,93 @@
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.IO;
+using Newtonsoft.Json;
+using RegressionGames.StateActionTypes;
+using RegressionGames.Types;
+using UnityEngine;
+using Object = UnityEngine.Object;
+
+namespace RegressionGames.DataCollection
+{
+    
+    /**
+     * An RGDataCollection instance is responsible for coordinating all data collection features on behalf
+     * of Regression Games. It will collect the replay data, logs, screenshots, and then upload these items
+     * to Regression Games. Currently only supports screenshots, with more to come in an upcoming PR.
+     */
+    public class RGDataCollection
+    {
+
+        private readonly string _sessionName;
+        private GameObject _parent;
+
+        public RGDataCollection(GameObject parent)
+        {
+            // Name the session, and setup a temporary directory for all data
+            _sessionName = Guid.NewGuid().ToString();
+            _parent = parent;
+        }
+
+        public void CaptureScreenshot(long tick)
+        {
+            Debug.Log($"Captured screenshot at tick {tick}");
```

Comment:
These should all be using RGDebug at appropriate log levels for each message

Reply:
Good catch, fixed!